webrtc.html

```diff
@@ -2389,6 +2389,58 @@
           attribute is the track that is associated with this
           <code><a>RTCRtpSender</a></code> object.</p>
         </dd>
+
+        <dt>Promise&lt;void&gt; replaceTrack (MediaStreamTrack withTrack)</dt>
+
+        <dd>
+            <p>Attempts to replace the track being sent with another track
+            provided, without renegotiation.</p>
+
+            <p>When the <dfn id="dom-rtpsender-replacetrack"><code title=
+            "">replaceTrack()</code></dfn> method is invoked, the user agent MUST
+            run the following steps:</p>
+
+            <ol>
+              <li>
+                <p>Let <var>p</var> be a new promise.</p>
+              </li>
+
+              <li>
+                <p>Let <var>withTrack</var> be the argument to this method.</p>
+              </li>
+
+              <li>
+                <p>Run the following steps asynchronously:</p>
+              </li>
+              <ol>
+              <li>
+                <p>If <var>withTrack</var> is compatible with the sender's existing
```

Comment:
This seems like a run-on sentence. Consider making the steps into an explicit list.